src/app/services/mail.service.ts

```diff
@@ -0,0 +1,124 @@
+import dedent from 'dedent-js'
```

Comment:
is this a package for the pink panther theme song? dedent, dedent, dedent dedent dedent dedent dedent...